filebeat/input/redis/harvester.go

```diff
@@ -81,25 +81,32 @@ func (h *Harvester) Run() error {
 		return nil
 	default:
 	}
-	// Writes Slowlog get and slowlog reset both to the buffer so they are executed together
+	// Writes SLOWLOG GET, SLOWLOG RESET, and ROLE to the buffer so they are executed together
 	h.conn.Send("SLOWLOG", "GET")
```

Comment:
Looks like the linter is complaining about this code now, do you mind handling the errors returned by the Send calls? Thanks.